src/platform/forms-system/src/js/definitions/directDeposit.jsx

```diff
@@ -175,13 +145,34 @@ const schema = optionalFields => {
       ? optionalFields.declineDirectDeposit.schema
       : { type: 'boolean' };
   }
+
+  s.properties = {
+    ...s.properties,
+    'view:directDepositInfo': { type: 'object', properties: {} },
+    'view:bankInfoHelpText': { type: 'object', properties: {} },
+  };
```

Comment:
I don't see how this is any different from including the `view` properties up above in the original declaration for `s`.

Reply:
`directDepositUiSchema` comes from https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/platform/forms-system/src/js/definitions/directDeposit.jsx#L41
 
If I remove or null out `'ui:order'`s 

![Screen Shot 2021-04-05 at 3 57 53 PM](https://user-images.githubusercontent.com/1094999/113619979-afaa5a00-9627-11eb-9248-9ff96b2c7753.png)

then the `declineDirectDeposit` is not in the correct location.

![Screen Shot 2021-04-05 at 3 57 44 PM](https://user-images.githubusercontent.com/1094999/113619962-aa4d0f80-9627-11eb-816a-98dc72e9ad69.png)


The reason for this is that without an `'ui:order'` property the objects within `uiSchema` are rendered in the order in which the properties are added to the `schema`